### PR TITLE
Pr mavlink interface cleanup

### PR DESCRIPTION
**Pull request by EliaTarasov**
This pr provides some cosmetic code fixes.

### CODE REVIEW

File: include/gazebo_mavlink_interface.h [hunk 1]
```diff
@@ -153,7 +153,7 @@ class GazeboMavlinkInterface : public ModelPlugin {
     groundtruth_lon_rad(0.0),
     groundtruth_altitude(0.0),
     mavlink_udp_port_(kDefaultMavlinkUdpPort),
-    qgc_udp_port_(kDefaultMavlinkUdpPort),
+    qgc_udp_port_(kDefaultQGCUdpPort),
```
Comment (mhkabir) [inline include/gazebo_mavlink_interface.h:156]: This changes it to port 14550. I wonder how the original version with `kDefaultMavlinkUdpPort` (14560) functioned correctly.

Comment (mhkabir) [reply include/gazebo_mavlink_interface.h:156]: Actually - do we ever use this QGC UDP link? Can we delete it completely? The PX4 executable should be routing the packets in reality.

**Comment by dagar** [comment]
If you're interested in cosmetic code changes I would encourage you to implement some kind of automated and enforced formatting for this repo (astyle or clang-format).

**Comment by EliaTarasov** [comment]
@mhkabir It works because in every `sdf` file this setting is `<qgc_udp_port>14550</qgc_udp_port>`
And i think we still need this setting. It allows the most obvious way to connect QGC to PX4 instance.

File: include/gazebo_mavlink_interface.h (hunk 1 above)
Comment (TSC21) [reply include/gazebo_mavlink_interface.h:156]: No. This is required because of HITL.

**Review by mhkabir — APPROVED** [review]

**Pull request merged**